service/init.go

```diff
@@ -229,9 +229,10 @@ func (nssf *NSSF) Start() {
 	}
 
 	serverScheme := factory.NssfConfig.Configuration.Sbi.Scheme
-	if serverScheme == "http" {
+	switch serverScheme {
+	case "http":
 		err = server.ListenAndServe()
-	} else if serverScheme == "https" {
+	case "https":
```

Comment:
Do we have to handle the case when `serverScheme` is neither http nor https? (This comment applies to all NFs)